students/17th/team1/minjihuh/user/views.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from django.http   import JsonResponse, HttpResponse;
+from django.views  import View
+
+from user.models   import (
+    User
+)
+
+class SignInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            name          = data['name']
+            email         = data['email']
+            password      = data['password']
+            username      = data['username']
+            MINIMUM_PASSWORD_LEN = 8
```

Comment:
상수 값은 함수 내부 보다는 함수 바깥 혹은 클래스 바깥에 위치하는게 좋습니다.
```suggestion
            MINIMUM_PASSWORD_LENGTH = 8
```
이름도 위와 같은게 더 좋을것 같아요

Reply:
앗 지금 바로 수정하겠습니다! :-)